### DISCUSSION TITLE
Off topic: Moving django-parler to jazzband?

### DISCUSSION THREAD

**Discussion by boosh** [jrief/django-formset#167] [General]
Hey I see you've committed to django-parler in the past. Parler looks like the best translation library and I want to use it but it's unmaintained. 

Have you seen [this issue](https://github.com/django-parler/django-parler/issues/353) about moving it to jazzband? Is there any way you could contact the maintainer to see if that could be done?

Thanks

**Comment by jrief** [comment]
I have met Diederick many years ago the last time. I do not have the privileges to transfer thid repository to another owner. And I wouldn't even do it if I could so.

**Reply by boosh** [reply]
Of course. But maybe you could contact him if you have his details since he doesn't seem to reply any more on that repo?
